docs/source/install/upgrades.rst

```diff
@@ -193,11 +193,24 @@ v3.5
      sudo rpm -e --nodeps nodejs
      sudo yum upgrade st2chatops
 
+* |st2| has been updated to configure only TLSv1.2 and v1.3 on nginx. The default nginx ST2 configuration will need to be updated manually, and nginx restarted:
+
+  .. sourcecode:: bash
+
+     sudo sed -i.bak 's|ssl_protocols.*|ssl_protocols             TLSv1.2 TLSv1.3;|g' /etc/nginx/conf.d/st2.conf
+     sudo systemctl restart nginx
```

Comment:
BTW per https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-deb.sh#L633-L634
the original nginx file shipped with the packages (incl. updated 3.5.0) is located at `/usr/share/doc/st2/conf/nginx/st2.conf` if we want to reference it.